vm-mock/vm.js

```diff
@@ -279,10 +279,66 @@ export default function createMockedABI(
           const addressStorage = ledger.get(contractAddress).storage;
           if (addressStorage.has(k)) {
             addressStorage.delete(k);
+          } else {
+            console.log('key not found');
```

Comment:
Maybe it will panic in that scenaior. The [documentation](https://as-sdk.docs.massa.net/functions/Storage.del.html) is not clear on that.